### PR TITLE
Added enchancement options in ExcelToPDF

### LINKED ISSUE #720: Add `More enhancement options` in `Excel to PDF` : Password protect
### Steps to reproduce
- open `Excel to PDF` using navigation drawer/home page

### Expected behaviour
- Add `More enhancement options` section (similar to Merge PDF)
- Add option to create password protected PDF

### PR DESCRIPTION
**Pull request by rob729**
# Description

2zi51t

Fixes #720

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by rob729** [comment]
@vyankatesh24 Please review this PR.

File: app/src/main/java/swati4star/createpdf/util/ExcelToPDFAsync.java [hunk 1]
```diff
@@ -3,15 +3,21 @@
 import android.os.AsyncTask;
 
 import com.aspose.cells.FileFormatType;
+import com.aspose.cells.LoadOptions;
+import com.aspose.cells.PdfSaveOptions;
+import com.aspose.cells.PdfSecurityOptions;
+
 import com.aspose.cells.Workbook;
 
+
```
Comment (vyankatesh24) [inline app/src/main/java/swati4star/createpdf/util/ExcelToPDFAsync.java]: Remove this line.

File: app/src/main/res/layout/fragment_excelto_pdf.xml [hunk 2]
```diff
@@ -5,39 +5,83 @@
...
-        android:id="@+id/tv_excel_file_name_bottom"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_alignParentBottom="true"
-        android:gravity="center_horizontal"
-        android:padding="8dp"
-        android:text="@string/excel_tv_view_text"
-        android:visibility="gone"  />
+            android:orientation="vertical">
+
```
Comment (vyankatesh24) [inline app/src/main/res/layout/fragment_excelto_pdf.xml]: Remove extra blank lines.

File: app/src/main/java/swati4star/createpdf/util/ExcelToPDFAsync.java [hunk 3]
```diff
@@ -37,8 +45,19 @@ protected void onPreExecute() {
     @Override
     protected Void doInBackground(Void... voids) {
         try {
-            final Workbook workbook = new Workbook(mPath);
-            workbook.save(mDestPath, FileFormatType.PDF);
+            LoadOptions loadOptions = new LoadOptions(FileFormatType.XLSX);
```
Comment (vyankatesh24) [inline app/src/main/java/swati4star/createpdf/util/ExcelToPDFAsync.java]: Is the conversion (with password) working for `.xls` files ?

Comment (rob729) [reply app/src/main/java/swati4star/createpdf/util/ExcelToPDFAsync.java]: Yes, it is working.
Line 48 is not of any use and I am removing it in my latest commit.

**Comment by rob729** [comment]
@vyankatesh24 Can you tell me the position on the screen where you want the name of the selected file to be shown?

**Comment by vyankatesh24** [comment]
> @vyankatesh24 Can you tell me the position on the screen where you want the name of the selected file to be shown?

As it was earlier (extreme bottom)

**Comment by rob729** [comment]
@vyankatesh24 I have done the suggested changes.
Please review the PR.

**Comment by rob729** [comment]
@vyankatesh24 I have fixed the bug.
Please review the PR.

**Pull request merged**